genie-web/src/test/groovy/com/netflix/genie/web/agent/apis/rpc/v4/endpoints/GRpcJobKillServiceImplSpec.groovy

```diff
@@ -65,31 +66,39 @@ class GRpcJobKillServiceImplSpec extends Specification {
             getPersistenceService() >> this.persistenceService
         }
         this.service = new GRpcJobKillServiceImpl(dataServices, this.agentRoutingService, this.requestForwardingService)
+
+        this.serviceSpy = Spy(service);
```

Comment:
given this (I haven't used this feature so don't know) but do we still need the actual `service` as a member or can we make it a local variable within setup?

Reply:
I kept it in case other tests don't need a Spy, but I think I'll just keep the Spy, and whoever wants a regular object can expose it later.